src/dbnode/namespace/kvadmin/ns_admin_test.go

```diff
@@ -154,3 +155,29 @@ func TestAdminService_ResetSchema(t *testing.T) {
 	err = as.ResetSchema("ns1")
 	require.NoError(t, err)
 }
+
+func TestAdminService_Crud(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	store := mem.NewStore()
+	var nsRegKey = "nsRegKey"
+	as := NewAdminService(store, nsRegKey, func() string {return "first"})
+	require.NotNil(t, as)
+
+	require.NoError(t, as.Add("ns1", namespace.OptionsToProto(namespace.NewOptions())))
+	require.Error(t, as.Add("ns1", namespace.OptionsToProto(namespace.NewOptions())))
+	require.NoError(t, as.Set("ns1", namespace.OptionsToProto(namespace.NewOptions())))
+	require.Error(t, as.Set("ns2", namespace.OptionsToProto(namespace.NewOptions())))
+
+	nsOpt, err := as.Get("ns1")
```

Comment:
can we do an assert that the returned thing is correct?

Reply:
sure